vnext/Desktop.ABITests/React.Windows.Desktop.ABITests.vcxproj

```diff
@@ -53,8 +53,13 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup>
     <IncludePath>$(ReactNativeWindowsDir);$(IncludePath)</IncludePath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)'=='Debug'" Label="Configuration">
```

Comment:
Consider putting these in React.cpp.props to share between projects.

Reply:
fixed, got to clean up a bunch of others as well. There are now a bunch of others proops that can and should be shared too. I'll keep that for another future cleanup.